doc/docs.md

```diff
@@ -2107,6 +2107,23 @@ println(blue)
 
 Global variables are not normally allowed, so this can be really useful.
 
+#### Modules
```

Comment:
There is already a separate Modules section, adding this breaks links.
Perhaps rename it to something like this: `#### Constants in modules` ?

Reply:
fixed - reformatted with bold - should good for now

I suggest regarding TOC double link targets:
1. use the existing doc check to create the TOC dynamical based on existing headlines and replace the TOC 
2. use `<div style="column-count: 3">TOC</div>` and not a table - if css moves from html to css or into its own style tag we could make it 1 column on mobile
3. fail on doc check wenn TOC creation detects multiple targets

Reply:
multiple TOC target check is implemented https://github.com/vlang/v/pull/10417
making the TOC is now very easy :-)

Reply:
div with style is not supported by GitHub markdown. 
we need to stay with the table